metrics/storage/fio-k8s/fio-test-ci.sh

```diff
@@ -28,7 +28,7 @@ function main() {
 
 	pushd "${FIO_PATH}"
 		echo "INFO: Running K8S FIO test"
-		sudo -E make test-ci
+		make test-ci
```

Comment:
Why sudo is not needed anymore?  Or it was never needed?

Reply:
@fidencio it was never needed it

Reply:
@fidencio we do not need sudo to run the test